### PR TITLE
Add script for scalability test

### PR DESCRIPTION
**Pull request by PhanLe1010**
longhorn/longhorn#2598

### CODE REVIEW

**Comment by innobead** [comment]
@meldafrawi please help review this one, and we will plan to have this automated in the future.

File: scalability_test/terraform/variables.tf [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+variable "aws_availability_zone" {
+  type    = string
+  default = "us-west-2a"
+}
+
+variable "distro_version" {
+  type    = string
+  default = "18.04"
```
Comment (meldafrawi) [inline scalability_test/terraform/variables.tf:23]: should we use 20.04 here ?

**Review by meldafrawi — COMMENTED** [review]
**LGTM** in general, left one question about Ubuntu AMI version.

File: scalability_test/terraform/variables.tf (hunk 1 above)
Comment (PhanLe1010) [reply scalability_test/terraform/variables.tf]: Fixed. Thank you

**Comment by PhanLe1010** [comment]
This one is ready for merging

**Review by joshimoo — APPROVED** [review]
Great Work at @PhanLe1010 
Since @meldafrawi already gave his LGTM, I am approving & merging this.
We also saw the scripts in action during the scalability issue analysis :)

**Pull request merged**